networking/src/Mockable/Concurrent.hs

```diff
@@ -121,13 +122,14 @@ data Async m t where
     AsyncThreadId :: Promise m t -> Async m (ThreadId m)
     Race :: m t -> m r -> Async m (Either t r)
     Link :: Promise m t -> Async m ()
+    UnsafeUnmask :: m a -> Async m a
```

Comment:
What's the story about `UnsafeUnmask`? Why did you add it?

Reply:
It was a part of `Mockable Bracket` before, and it is used in some of `Mockable Async` combinators.

Reply:
Ok.